### PR TITLE
PRS Wand Bawasan Susenyos

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New record, edits

### CODE REVIEW

File: PRS10001-11000/PRS10025WandBa.xml [hunk 1]
```diff
@@ -44,14 +44,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
-               <persName>
-                  
-                  Wänd Bäwäsän
-               </persName>
-               <occupation> däǧǧ.  </occupation>
-            </person>
+           <person sex="1">
+               <persName xml:lang="gez" xml:id="n1">ወንድ፡ በወሰን፡</persName>
+               <persName xml:lang="gez" type="normalized" corresp="#n1">Wand Bawasan</persName>
+               <persName type="alt"><roleName type="title">daǧǧ azmāč</roleName>Wand Bawasan</persName>
```
Comment (eu-genia) [inline PRS10001-11000/PRS10025WandBa.xml:50]: space missing between title and name

File: new/PRS12945Susenyos.xml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        <revisionDesc>
+            <change who="DN" when="2020-04-02">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">Susenyos</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1"/>
```
Comment (eu-genia) [inline new/PRS12945Susenyos.xml:49]: please fix fidel/transcription (when creating new files through the app please always first edit them and only then commit)

File: new/PRS12945Susenyos.xml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                    <faith type="EOTC"/>
+                    <occupation type="ruler"/>
+                    <nationality type="Ethiopia"/>
+                    <birth/>
+                    <death/>
+                    <floruit/>
+                </person>
+                <listRelation>
+                    <relation name="lawd:hasAttestation" active="PRS12945Susenyos" passive=""/>
```
Comment (eu-genia) [inline new/PRS12945Susenyos.xml:58]: relation passive - again please first edit and then commit, don't push files just as they were produced by the app

File: new/PRS12945Susenyos.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/PRS12945Susenyos.xml:49]: OK

File: new/PRS12945Susenyos.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/PRS12945Susenyos.xml:58]: Yes, sure.

File: PRS10001-11000/PRS10025WandBa.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply PRS10001-11000/PRS10025WandBa.xml:50]: Corrected.

File: new/PRS12945Susenyos.xml [hunk 4]
```diff
@@ -1,11 +1,11 @@
-<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
-schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
 type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 <TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS12945Susenyos" xml:lang="en" type="pers">
```
Comment (eu-genia) [inline new/PRS12945Susenyos.xml:4]: This person already has an ID, PRS9039Susenyos. Doublet

Comment (DenisNosnitsin1970) [reply new/PRS12945Susenyos.xml:4]: Indeed, no idea how I missed him.

**Pull request merged**